docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -150,67 +150,109 @@ See the individual entry for the function.
 == Datetime Functions
 
 These functions use either a datetime value expression as an argument or
-return a result of datetime data type:
+return a result of DATETIME, VALUE or CHAR:
```

Comment:
The old text was correct. It categorized datetime functions as being those that either accept datetimes or produce them. The new change, though, would allow SUBSTRING as a Datetime function (since it produces a CHAR output).

Reply:
@DaveBirdsall Thanks for your explanations, I've corrected. :)